dev-games/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A Simulator Construction Toolkit"
+DESCRIPTION="Set of open-source libraries designed as building blocks \
+for quickly assembling 3D simulations, games, and visualization applications."
+HOMEPAGE="https://home.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
+PATCHES="simgear-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion
+	lib:libsimgear$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost$secondaryArchSuffix
```

Comment:
@kenmays, it looks like you have built SimGear without haikuporter, then wrote the recipe, but did not test it...
What makes me believe this is that there are no `lib:libboost` (nor `devel:libboost`).
You need to use the exact names of these libs, e.g. `libboost_filesystem`, `libboost_timer`, `libboost_thread`.
There are lots of these `libboost_foobar` libs, and you need to identify which are needed.
The trick: Start pretending it needs one of these libs (so that haikuporter will pull one of the boost_devel packages), then build the recipe, and then use `objdump -x libsimgear.so | grep NEEDED` to identify which libs are really needed.

I've already made a few cosmetic fixes to your recipe and also regenerated the patchset.

Can you try to fix the `BUILD_REQUIRES` and `REQUIRES` so that the recipe can be tested with haikuporter?
Thanks!

Reply:
I fixed the recipe but notice you updated it already. Great!